pages/admin/users/index/invite.vue

```diff
@@ -56,7 +57,7 @@ export default {
       return this.generatedInvitation ? hostname + "/user/register/" + this.generatedInvitation._id : "";
     },
     hostname() {
-      return process.env.BASE_URL;
+      return this.$config.BASE_URL;

```

Comment:
I'd much prefer the way to access the env remain consistent everywhere, using multiple syntaxes like this could get confusing.
```suggestion
      return process.env.BASE_URL;
```